.github/workflows/pr-ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
       # Same settings as publish
       run-playwright-with-skip-grafana-dev-image: true
-      run-playwright-with-grafana-dependency: '>=12.0.0'
+      run-playwright-with-grafana-dependency: '>=12.1.0'
```

Comment:
Unfortunately 'latest' didn't work, but in the future we can increment to the latest version it's pulling like this.  By using `>=` we'll still auto-update and break on the next highest version, then need to update our workflow again.